### PR TITLE
SL-553: Add MCOE triage form and visit action

### PR DESCRIPTION
**Pull request by cioan**
Added MCOE Triage visit action.

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/pihcore/apploader/CustomAppLoaderConstants.java [hunk 1]
```diff
@@ -148,6 +148,7 @@ public static final class Apps {
     public static final class Extensions {
 
         public static final String CHECK_IN_VISIT_ACTION = "pih.checkin.visitAction";
+        public static final String TRIAGE_VISIT_ACTION = "pih.triage.visitAction";
```
Comment (lnball) [inline api/src/main/java/org/openmrs/module/pihcore/apploader/CustomAppLoaderConstants.java:151]: There are other triage_visits and this one seems specific for mch triage.  If you don't think it's needs, I'm fine with that @cioan

**Review by lnball — COMMENTED** [review]
See comment

**Review by mogoodrich — APPROVED** [review]
One small comment, but generally LGTM, thanks @cioan !

File: api/src/main/java/org/openmrs/module/pihcore/apploader/CustomAppLoaderConstants.java (hunk 1 above)
Comment (mogoodrich) [reply api/src/main/java/org/openmrs/module/pihcore/apploader/CustomAppLoaderConstants.java:151]: Yes, agreed, that  including "MCOE" in the name males sense.

Comment (mogoodrich) [reply api/src/main/java/org/openmrs/module/pihcore/apploader/CustomAppLoaderConstants.java:151]: Or perhaps "maternal" or "MCH" triage if we want something more generic?

File: api/src/main/java/org/openmrs/module/pihcore/apploader/CustomAppLoaderFactory.java [hunk 2]
```diff
@@ -231,6 +231,10 @@ private void loadAppsAndExtensions() throws UnsupportedEncodingException {
             enableCheckIn(config);
         }
 
+        if (config.isComponentEnabled(Components.MCOE_TRIAGE)) {
```
Comment (mogoodrich) [inline api/src/main/java/org/openmrs/module/pihcore/apploader/CustomAppLoaderFactory.java:234]: I'm wondering if we want to name this something like "maternal triage" or "mch triage"?

**Review by mogoodrich — APPROVED** [review]
Actually, see my updated thoughts... I'm wondering if we want to try to use "MCH" and/or "maternal" as much as possible so we don't tie the naming convention to a specific facility in the specific country?  I guess it depends on how much we think this form is MCOE-specific (would we ever use it at Wellbody?)

File: api/src/main/java/org/openmrs/module/pihcore/apploader/CustomAppLoaderConstants.java (hunk 1 above)
Comment (cioan) [reply api/src/main/java/org/openmrs/module/pihcore/apploader/CustomAppLoaderConstants.java]: Thanks! I have renamed to MCH triage.

File: api/src/main/java/org/openmrs/module/pihcore/apploader/CustomAppLoaderFactory.java (hunk 2 above)
Comment (cioan) [reply api/src/main/java/org/openmrs/module/pihcore/apploader/CustomAppLoaderFactory.java]: Thanks!

**Pull request merged**